Extension/src/LanguageServer/client.ts

```diff
@@ -3513,12 +3514,23 @@ export class DefaultClient implements Client {
                 }
                 lastEdit = new vscode.TextEdit(range, edit.newText);
                 const position: vscode.Position = new vscode.Position(edit.range.start.line - editPositionAdjustment, edit.range.start.character);
+                // If the command was invoked via the copy command, copy the declaration or definition to clipboard.
+                if (copy) {
+                    vscode.env.clipboard.writeText(edit.newText).then(() => {
+                        // Writing was successful
+                    }, () => {
+                        // Writing was unsuccessful
+                        util.promptCDDFailed(true);
```

Comment:
I think this is the only case where create declaration or definition failed for an explicit request to copy needs to be handled.